### PR TITLE
west.yml: update hal stm32xx modules

### PR DESCRIPTION
**Pull request by FRASTM**
This updates the stm32cube/stm32xx

Requires https://github.com/zephyrproject-rtos/hal_stm32/pull/47

Signed-off-by: Francois Ramu <francois.ramu@st.com>

### CODE REVIEW

**Comment by FRASTM** [comment]
change to
revision: /pull/47/head

**Comment by FRASTM** [comment]
rebase

File: west.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ manifest:
       revision: fa481784b3c49780f18d50bafe00390ccb62b2ec
       path: modules/hal/st
     - name: hal_stm32
-      revision: dee4413253623ec575bb8b58abd56f91afe903bb
+      revision: pull/47/head
```
Comment (erwango) [inline west.yml]: @FRASTM, https://github.com/zephyrproject-rtos/hal_stm32/pull/47 is now merged.
Can you update the SHA1 ?

**Comment by FRASTM** [comment]
set the SHA1 after merge of stm32cube versions (modules/hal/stm32)

**Review by erwango — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Pull request merged**